src/components/AuthLayout.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Form, NavLink } from 'react-router-dom';
+import PropTypes, { bool } from 'prop-types';
```

Comment:
You can remove the bool here, it's currently not being used. Just leave as PropTypes